azure_monitor/src/azure_monitor/sdk/auto_collection/live_metrics/sender.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+import json
+import logging
+import time
+
+import requests
+
+from azure_monitor.protocol import LiveMetricEnvelope
+
+DEFAULT_LIVEMETRICS_ENDPOINT = "https://rt.services.visualstudio.com"
+LIVE_METRICS_TRANSMISSION_TIME_HEADER = "x-ms-qps-transmission-time"
+
+logger = logging.getLogger(__name__)
+
+
+class LiveMetricsSender:
+    """Live Metrics Sender
+
+    Send HTTP requests to Live Metrics service
+    """
+
+    def __init__(self, instrumentation_key: str):
+        self._endpoint = DEFAULT_LIVEMETRICS_ENDPOINT
+        self._instrumentation_key = instrumentation_key
+
+    def ping(self, envelope: LiveMetricEnvelope):
+        return self._send_request(json.dumps(envelope.to_dict()), "ping")
+
+    def post(self, envelope: LiveMetricEnvelope):
+        return self._send_request(json.dumps([envelope.to_dict()]), "post")
+
+    def _send_request(self, data: str, request_type: str) -> requests.Response:
+        try:
+            url = "{0}/QuickPulseService.svc/{1}?ikey={2}".format(
+                self._endpoint, request_type, self._instrumentation_key
+            )
+            response = requests.post(
+                url=url,
+                data=data,
+                headers={
+                    "Expect": "100-continue",
```

Comment:
Do we need `Content-Length: 247` and `Connection: Keep-Alive` as well as defined by [the spec](https://msazure.visualstudio.com/One/_git/Mgmt-AppInsights-LiveMetricsService?path=%2Fdocs%2Fsdk.api.md&version=GBmaster&_a=preview&anchor=request?

Reply:
Added automatically by requests library